iceoryx_examples/callbacks/README.md

```diff
@@ -184,6 +194,113 @@ Afterward, we reset both caches to start fresh again.
     }
 ```
 
+### ice_callbacks_listener_as_class_member.cpp
```

Comment:
this is actually more powerful than just a listener as class member. You can pass virtually everything as pointer to additional context data to the callback, being an integer, mutex or instance of a class